index.html

```diff
@@ -231,6 +231,6 @@ <h2 class="success__title">Изображение успешно загруже
   <template id="messages">
     <div class="img-upload__message  img-upload__message--loading">Загружаем...</div>
   </template>
-
+  <script srs="js/functions.js"></script>
```

Comment:
Ошибка в имени аттрибута, нужено `src` от слова `source` 

```suggestion
  <script src="js/functions.js"></script>
```